src/hotspot/share/opto/superword.cpp

```diff
@@ -91,6 +91,8 @@ SuperWord::SuperWord(PhaseIdealLoop* phase) :
 #endif
 }
 
+static const bool _do_vector_loop_experimental = false; // Experimental vectorization which uses data from loop unrolling.
```

Comment:
For such purposes, I find debug VM flags a better option: they are constant in product binaries, easy to experiment/change from command-line with debug binaries, and enumerated in a single place.

Any reason to prefer `static const` declarations?

Reply:
Adding any flag is exception - we should avoid it if it is not needed immediately. We have a lot of flags already.
As I said currently this code is broken - I want to remove it from debug builds too.
And it is easy to modify this value later - make it a field with the same name and initialize with diagnostic UseNewCode flag value as I did for testing.